build.gradle.kts

```diff
@@ -33,14 +34,21 @@ val dockerComposePluginVersion = "0.9.4"
 val junitVersion = "5.5.1"
 
 dependencies {
-  implementation(localGroovy())
+  implementation(kotlin("stdlib"))
   implementation("com.avast.gradle:gradle-docker-compose-plugin:$dockerComposePluginVersion")
 
   testImplementation(gradleTestKit())
   testImplementation("org.junit.jupiter:junit-jupiter:$junitVersion")
 }
 
 tasks {
+  withType<KotlinCompile> {
+    kotlinOptions {
+      freeCompilerArgs = listOf("-Xjsr305=strict")
+      jvmTarget = JavaVersion.VERSION_11.toString()
```

Comment:
I think we should keep compatibility with JDK 8 users.

Reply:
done